utils/saasherder.py

```diff
@@ -389,15 +389,19 @@ def _collect_images(self, resource):
         return images
 
     @staticmethod
-    def _check_image(image, image_patterns, image_auth, error_prefix):
+    @retry()
```

Comment:
we already retry 5x: https://github.com/app-sre/sretoolbox/blob/master/sretoolbox/container/image.py#L276

Reply:
if you want to retry more, you can just increase that number ^

Reply:
but does it handle `ConnectionError` like observed here?
https://ci.int.devshift.net/job/openshift-saas-deploy-notifications-insights-stage/8/console

Reply:
also, would this make sense?
https://github.com/app-sre/sretoolbox/pull/25

Reply:
> but does it handle ConnectionError like observed here?

no

> also, would this make sense?

yes!